### ISSUE TITLE
piping "osc get pods" to awk should be replaced with "osc status"

### ISSUE THREAD

**Issue by brenton** [openshift/training#226]
In general I think `osc status` provides better unmodified output.  It doesn't show all the same information but given how verbose and hard to parse `osc get pods` is I think it should only be used for debugging purposes or for passing it to something that can handle json or yaml.

`osc status` will mention if the deployment succeeded or not.  If it said the deployment succeeded but the pod still wasn't responding I might use `osc get pods` to check for an error message.

**Comment by thoraxe** [comment]
osc status still doesn't provide sufficient information in all cases. Like, it doesn't even show the deployed pods(s)' name...
